src/axom/inlet/Field.cpp

```diff
@@ -285,5 +285,23 @@ std::shared_ptr<Field> Field::validValues(const std::initializer_list<double>& s
   return validValues(std::vector<double>(set));
 }
 
+std::shared_ptr<Field> Field::registerVerifier(std::function<bool()> lambda) {
+  verifier = lambda;
+  return shared_from_this();
+}
+
+bool Field::verify() {
+  if (verifier && !verifier()) {
+    SLIC_WARNING(fmt::format("Field {0} failed verification", 
+                 m_sidreGroup->getPathName()));
+    return false;
+  }
+  return true;
+}
+
+std::string Field::getName() {
+  return m_sidreGroup->getName();
```

Comment:
This exposes a need for another helper function.  This name isn't enough but the getPathName is probably too much.

https://axom.readthedocs.io/en/develop/doxygen/html/classaxom_1_1sidre_1_1Group.html#a7a4cb38dfb777394b1e60d66ec5158bb

Can you create a function called something like getFullName in the new helper function files that takes a Field or a Table and returns the sidre path name minus the root groups name?

So basically if Inlet's root group is "something/inlet" and the path name of this sidre group/view is "somethign/inlet/sometable/somefield" it would just return "sometable/somefield".